Gordon360/ApiControllers/JobsController.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Collections.Generic;
+using System.Web.Http;
+using System.Security.Claims;
+using Gordon360.Exceptions.ExceptionFilters;
+using Gordon360.Repositories;
+using Gordon360.Static.Methods;
+using Gordon360.Models.ViewModels;
+using Gordon360.Services.ComplexQueries;
+using Gordon360.Services;
+using Newtonsoft.Json.Linq;
+using Gordon360.AuthorizationFilters;
+using Gordon360.Static.Names;
+
+namespace Gordon360.ApiControllers
+{
+    [Authorize]
+    [CustomExceptionFilter]
+    [RoutePrefix("api/jobs")]
+    public class JobsController : ApiController
+    {
+        private IJobsService _jobsService;
+        private IAccountService _accountService;
+
+        public JobsController()
+        {
+            IUnitOfWork _unitOfWork = new UnitOfWork();
+            _jobsService = new JobsService(_unitOfWork);
+            _accountService = new AccountService(_unitOfWork);
+        }
+
+        /// <summary>
```

Comment:
Do you need this still?

Reply:
We've removed that at least three separate times, no idea why it keeps showing up. It won't hurt to have it in there for now, since we'll remove it (again) in a future commit.